vignettes/getting-started.Rmd

```diff
@@ -113,7 +115,10 @@ To begin modeling, first create a model object. This is an S3 object with an acc
 The `.yaml_path` argument should have the same name (but with a `.yaml` extension) as your base model control stream. For instance, the command below assumes you have a control stream name `1.ctl` or `1.mod` in the model directory you just defined. The `.description` argument is required and corresponds to what would be in the `$PROB` definition in a control stream.
 
 ```{r first model}
-mod1 <- new_model(.yaml_path = "1.yaml", .description = "our first model")
+mod1 <- new_model(
+  system.file("nonmem/1", package = "rbabylon"), 
```

Comment:
I feel strongly this should look like a pattern that a user (preferably a Metrum user) would use in real code. Certainly that would _not_ be `system.file`.

I guess this is going to change once we actually deprecate the `set_model_directory()`, so we can cross that bridge when we come to it... can you put a TODO in a vignette? If so, let's do that here. Or actually maybe just put a comment in here about it https://github.com/metrumresearchgroup/rbabylon/issues/165

Reply:
This change is a consequence of deprecating the `.directory` argument to `new_model()`. Now, the `.path` argument needs to indicate the absolute model path. `system.file()` is a way to (reliably) find files in packages using the structure of the installed package on disk, but I agree it may not be user-oriented. We could instead use

```r
mod1 <- new_model(file.path(MODEL_DIR, "1"), .description = "our first model")
```

which might be clearer (though I'm not sure we want to have language that encourages users to think of a "model directory").